crates/rspack_plugin_css/src/dependency/local_ident.rs

```diff
@@ -95,3 +96,105 @@ impl DependencyTemplate for CssLocalIdentDependency {
 
 impl AsContextDependency for CssLocalIdentDependency {}
 impl AsModuleDependency for CssLocalIdentDependency {}
+
+#[derive(Debug, Clone)]
+pub struct CssUseLocalIdentDependency {
+  id: DependencyId,
+  local_ident: String,
+  convention_names: Vec<String>,
+  start: u32,
+  end: u32,
+}
+
+impl CssUseLocalIdentDependency {
+  pub fn new(local_ident: String, convention_names: Vec<String>, start: u32, end: u32) -> Self {
+    Self {
+      id: DependencyId::new(),
+      local_ident,
+      convention_names,
+      start,
+      end,
+    }
+  }
+}
+
+impl Dependency for CssUseLocalIdentDependency {
+  fn id(&self) -> &DependencyId {
+    &self.id
+  }
+
+  fn category(&self) -> &DependencyCategory {
+    &DependencyCategory::CssLocalIdent
+  }
+
+  fn dependency_type(&self) -> &DependencyType {
+    &DependencyType::CssLocalIdent
+  }
+
+  fn get_exports(&self, _mg: &rspack_core::ModuleGraph) -> Option<ExportsSpec> {
+    Some(ExportsSpec {
+      exports: ExportsOfExportsSpec::Array(
+        self
+          .convention_names
+          .iter()
+          .map(|name| {
+            ExportNameOrSpec::ExportSpec(ExportSpec {
+              name: name.as_str().into(),
+              can_mangle: Some(false),
+              ..Default::default()
+            })
+          })
+          .collect(),
+      ),
+      ..Default::default()
+    })
+  }
+
+  fn could_affect_referencing_module(&self) -> rspack_core::AffectType {
+    rspack_core::AffectType::False
+  }
+
+  fn get_referenced_exports(
+    &self,
+    _module_graph: &rspack_core::ModuleGraph,
+    _runtime: Option<&RuntimeSpec>,
+  ) -> Vec<rspack_core::ExtendedReferencedExport> {
+    println!("111111");
+
+    vec![ExtendedReferencedExport::Array(vec![self
```

Comment:
This doesn't seem to be called at all.

Reply:
This should be called, this is a bit tricky that this dependency should bind to `SelfModuleFactory`, this may require some familiarity with how rspack's tree shaking is implemented, so I will take over this PR and fix it, thanks for the contribution